osquery/core/system.cpp

```diff
@@ -131,7 +131,13 @@ Status checkStalePid(const std::string& content) {
       return Status(1, "Error querying processes: " + q.getMessageString());
     }
 
-    if (q.rows().size() >= 1 && q.rows().front()["name"] == "osqueryd") {
+    bool is_osqueryd = false;
```

Comment:
I like the `[]` operator, but `const`ing rows makes more sense. I'm happy with using the const-map accessors :)